src/reducer.ts

```diff
@@ -161,28 +161,80 @@ type ReducerCreator = {
    * @returns A registered reducer that can be passed into `combineReducers`, or
    *          called directly as if it was the `reducer` parameter itself.
    */
-  <State>(
-    actionCreator: UnknownActionCreator,
-    reducer: Reducer<State, VoidPayload>
-  ): RegisteredReducer<State, VoidPayload>;
+  <State, Payload = VoidPayload>(
+    actionCreator: UnknownActionCreator | UnknownActionCreator[],
```

Comment:
The code is quite clear, so I was comfortable making this change.

Reply:
Note, I've removed this option in the new `actionReducer` because:
- it was hardly if ever used in ardoq-front
- it was very annoying to type correctly, mostly forcing us to use `any`